config/cijenkinsio/jenkins-ci-default.yaml

```diff
@@ -0,0 +1,443 @@
+                      scope: GLOBAL
+                      id: "jenkins-dockerhub"
+                      fileName: "config.json.zip"
+                      secretBytes: ${DOCKER_HUB_CONFIG_BYTES} # DOCKER_HUB_CONFIG_BYTES="$(cat ~/.docker/config.json | base64)"
+                      description: Jenkins account used to publish on dockerhub jenkins4eval organisation
+        tools: |
+          tool:
+            git:
+              installations:
+              - name: "jgit"
+            groovyInstallation:
+              installations:
+              - name: "groovy"
+                properties:
+                - installSource:
+                    installers:
+                    - groovyInstaller:
+                        id: "2.4.7"
+            jdk:
+              installations:
+              - name: "jdk8"
+                properties:
+                - installSource:
+                    installers:
+                    - zip:
+                        label: "linux"
+                        subdir: "jdk-8u242-b08"
+                        url: "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u242-b08/OpenJDK8U-jdk_x64_linux_hotspot_8u242b08.tar.gz"
+                    - zip:
+                        label: "windows"
+                        subdir: "jdk-8u242-b08"
+                        url: "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u242-b08/OpenJDK8U-jdk_x64_windows_hotspot_8u242b08.msi"
+              - name: "jdk11"
+                properties:
+                - installSource:
+                    installers:
+                    - zip:
+                        label: "linux"
+                        subdir: "jdk-11.0.3+7"
+                        url: "https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.3%2B7/OpenJDK11U-jdk_x64_linux_hotspot_11.0.3_7.tar.gz"
+                    - zip:
+                        label: "windows"
+                        subdir: "jdk-11.0.3+7"
+                        url: "https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.3%2B7/OpenJDK11U-jdk_x64_windows_hotspot_11.0.3_7.zip"
+            maven:
+              installations:
+              - name: "mvn"
+                properties:
+                - installSource:
+                    installers:
+                    - maven:
+                        id: "3.5.4"
+            groovyInstallation:
+              installations:
+              - name: "groovy"
+                properties:
+                - installSource:
+                    installers:
+                    - groovyInstaller:
+                        id: "2.4.7"
+        jobs-settings: |
+          jobs:
+            - script: >
+                def configuration = [
+                  [
+                    name        : "Core",
+                    repositories: "jenkins"
+                  ],
+                  [
+                    name        : "Infra",
```

Comment:
thoughts on moving this folder to infra.ci.jenkins.io?
the main reason for that is there's a number of credentials that are only on the infra folder and jcasc and folder level credentials don't work very well as far as i know.

it also reduces any risk as the credentials that this has will be very limited then.